### ISSUE TITLE
Follow up - Astronomy

### ISSUE THREAD

**Issue by dmpr0** [osmandapp/OsmAnd-iOS#5578]
# Configure view
https://github.com/osmandapp/OsmAnd-Issues/issues/3254#issuecomment-5092841761

<img width="235" height="225" alt="Image" src="https://github.com/user-attachments/assets/6134843e-6451-440f-919e-2551896ea799" />

- [x] Change button paddings 

## Accessibility 

**Map button labels**
- [x] Configure screen

### Configure view

- [x] Close button doesn't respond to screen reader
- [x] Big buttons doesn't work with screen reader

# Explore
https://github.com/osmandapp/OsmAnd-Issues/issues/3255#issuecomment-5068962571

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/dc8e8912-ed54-4dc9-a40a-527e9ad50d39"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Taping Back on iPad should act as back button, not as close Search button

- - - - - -
 
<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/83a1e4bc-69f6-43ee-baaf-9216a8f0ec04"  width="300" height="auto"/> </td>
      </tr>
    </table>
</details>  

- [x] Speed up the search bar’s appearance. 
- [x] Make the search button reappear faster.  

# Map / Plugin
https://github.com/osmandapp/OsmAnd-Issues/issues/3230

- [x] Use string for 
```
"north_abbreviation" = "N";
"south_abbreviation" = "S";
"east_abbreviation" = "E";
"west_abbreviation" = "W";
```

# Testing 1

> OsmAnd Maps 5431, released 23.07.26

**Severity**: 🟠 Major / **Priority**: 🟠 Normal

* [x] 1. After activating the **Red** filter, AR does not display anything.

<details><summary>Details</summary>
<table style="width:100%">
  <tr>
    <td><video source src="https://github.com/user-attachments/assets/2053f191-8073-44e4-8bf4-500cc59bfc6a" width="300" height="auto"/></td>
    <td><video source src="https://github.com/user-attachments/assets/525361a0-636d-499f-b1d7-a8c7bdb78c24" width="300" height="auto"/></td>
  </tr>
</table>
</details>

**Severity**: 🟡 Minor / **Priority**: 🟠 Normal

* [x] 1. If **Compass direction** is enabled, opening AR displays **Compass direction**, but closing AR displays **Manual mode**.

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/18ba1e1a-23b7-46b5-9077-6677f8c35699"></video>
</details>

* [x] 2. When opening **Configure view**, all other controls disappear.

<details><summary>Details</summary>
<table style="width:100%">
  <tr>
    <td><video source src="https://github.com/user-attachments/assets/01fdfe1f-75f9-464e-9d62-03dc278fc062" width="300" height="auto"/></td>
    <td><video source src="https://github.com/user-attachments/assets/183fd698-4a3d-4cfa-a471-7f092484ae3b" width="300" height="auto"/></td>
  </tr>
</table>
</details>

# Context menu

<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/7c1d5afa-10ac-48a9-be86-1f8cbf288cc2"  width="300" height="auto"/> </td>
  </tr>
</table>   

* We need to add an explanation for missing rise or set times.  This information block should inform the user that the object is always below the horizon and therefore not visible from this location or it’s always above the horizon and always up.

# Eclipses 

Discover a way to add solar and lunar eclipses.
